company/okrs/2020_q1.md

```diff
@@ -0,0 +1,103 @@
+# 2020-Q1 OKRs
+
+## CEO
+
+1. **CEO: Grow ARR consistently**
+   1. [$N][N] net new ARR
+   1. Sales leadership owns revenue targets and hiring plan
+   1. Sales, engineering, and product have a working process for prioritization and delivery
+1. **CEO: Aspirational, efficient all-remote team**
+   1. Roles and responsibilities are documented and maintained in handbook
+   1. Team members rely on the handbook for cross-functional info and processes
+   1. The handbook is fast to browse and search
+
+## Sales and marketing
+
+1. **Sales (Closing Deals): Achieve [$N][N] net new ARR.**
+   1. Have each AE close 3 Tier 1 deals (>=$100K ARR) and 10 non-Tier 1 deals, 2 of which are from their own outbound outreach efforts.
+   1. Get 70% of companies on a free trial to convert to paid customers.
+1. **Sales (Pipeline Optimization): Scale our sales pipeline to 50+ deals per AE, of which 60% of deals are engaged and actively evaluating Sourcegraph.**
+   1. Round robin new instances across team to collectively achieve a 15%+ response rate from Prospects who set up a new Sourcegraph instance.
+
+1. **VP Business: Build a Customer Success organization that supports and helps us exceed our ARR goals.**
```

Comment:
@dadlerj @juliasourceress This is pending our discussion of Customer Engineering

Reply:
👍 

@sqs in case you didn't see it, I updated my OKRs and left a note on this one in a separate PR at https://github.com/sourcegraph/about/pull/513 (apologies if I used the wrong process here?)